a100env.yaml

```diff
@@ -0,0 +1,26 @@
+name: testbench-peft
+dependencies:
+  - python=3.8
+  - pip
+  - pip:
+    - torch==2.1.0
```

Comment:
Isn't it supposed to be 1.13.1? Why is it 2.1.0?

Reply:
Yeah, sorry about that. I might have mistakenly changed the version to 2.1.0 for another task.